src/assets/images/discord-icon-blue.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="29" height="32" viewBox="0 0 29 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M18.8416 15.2C18.8416 16.176 18.1216 16.976 17.2096 16.976C16.3136 16.976 15.5776 16.176 15.5776 15.2C15.5776 14.224 16.2976 13.424 17.2096 13.424C18.1216 13.424 18.8416 14.224 18.8416 15.2ZM11.3696 13.424C10.4576 13.424 9.73763 14.224 9.73763 15.2C9.73763 16.176 10.4736 16.976 11.3696 16.976C12.2816 16.976 13.0016 16.176 13.0016 15.2C13.0176 14.224 12.2816 13.424 11.3696 13.424ZM28.2656 3.296V32C24.2347 28.4379 25.5239 29.617 20.8416 25.264L21.6896 28.224H3.54562C1.73763 28.224 0.265625 26.752 0.265625 24.928V3.296C0.265625 1.472 1.73763 0 3.54562 0H24.9856C26.7936 0 28.2656 1.472 28.2656 3.296ZM23.7056 18.464C23.7056 13.312 21.4016 9.136 21.4016 9.136C19.0976 7.408 16.9056 7.456 16.9056 7.456L16.6816 7.712C19.4016 8.544 20.6656 9.744 20.6656 9.744C16.8649 7.66094 12.4004 7.66056 8.71362 9.28C8.12162 9.552 7.76963 9.744 7.76963 9.744C7.76963 9.744 9.09762 8.48 11.9776 7.648L11.8176 7.456C11.8176 7.456 9.62563 7.408 7.32163 9.136C7.32163 9.136 5.01762 13.312 5.01762 18.464C5.01762 18.464 6.36162 20.784 9.89762 20.896C9.89762 20.896 10.4896 20.176 10.9696 19.568C8.93763 18.96 8.16962 17.68 8.16962 17.68C8.405 17.8447 8.79313 18.0583 8.82563 18.08C11.5263 19.5924 15.3624 20.0879 18.8096 18.64C19.3696 18.432 19.9936 18.128 20.6496 17.696C20.6496 17.696 19.8496 19.008 17.7536 19.6C18.2336 20.208 18.8096 20.896 18.8096 20.896C22.3456 20.784 23.7056 18.464 23.7056 18.464Z" fill="#0D38D3"/>
```

Comment:
instead of saving multiple colors of one icon, is there a way we can have a svg that allows color to be passed

looks like `fill="#0D38D3"` might help?